lib/ueberauth/strategy/google.ex

```diff
@@ -124,7 +131,7 @@ defmodule Ueberauth.Strategy.Google do
     resp = Ueberauth.Strategy.Google.OAuth.get(token, path)
 
     case resp do
-      {:ok, %OAuth2.Response{status_code: 401, body: _body}} ->
```

Comment:
Does a 401 here return with an `:error` in the tuple? I think when I was testing it it still responded with an `:ok`

Reply:
The latest `oauth2` version has changed so that any successful HTTP status codes (200..399) returns `{:ok, resp}` and all others returns `{:error, resp}`.